src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud/new.jelly

```diff
@@ -2,16 +2,14 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form">
   <l:layout permission="${app.SYSTEM_READ}" title="${%New pod template}">
     <j:set var="readOnlyMode" value="${!app.hasPermission(app.ADMINISTER)}"/>
-        <j:set var="instance" value="${it.newTemplate}"/>
-        <j:set var="descriptor" value="${instance.descriptor}"/>
     <l:breadcrumb title="${%New pod template }"/>
     <st:include page="sidepanel.jelly" it="${it}"/>
     <l:main-panel>
       <h1>${%New pod template settings}</h1>
       <f:form method="post" action="create" name="config" class="jenkins-form">
         <!-- main body of the configuration -->
 
-        <st:include it="${instance}" page="config.jelly"/>
+        <st:include class="${it.templateDescriptor.clazz}" page="config.jelly"/>
```

Comment:
@Vlatombe that  leads to  

```
WARNING	h.ExpressionFactory2$JexlExpression#evaluate: Caught exception evaluating: h.filterDescriptors(it,attrs.descriptors) in /jenkins/manage/cloud/12/new. Reason: java.lang.NullPointerException: Descriptor list is null for context 'class org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud' in thread 'Handling GET /jenkins/manage/cloud/12/new from 127.0.0.1 : qtp219578974-17 KubernetesCloud/new.jelly PodTemplate/config.jelly'
java.lang.NullPointerException: Descriptor list is null for context 'class org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud' in thread 'Handling GET /jenkins/manage/cloud/12/new from 127.0.0.1 : qtp219578974-17 KubernetesCloud/new.jelly PodTemplate/config.jelly'
```

Reply:
when I switch back to `<st:include it="${instance}" page="config.jelly"/>` then there is only one error.

```
java.lang.NullPointerException: Descriptor list is null for context 'class org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud' in thread 'Handling GET /jenkins/manage/cloud/12/new from 127.0.0.1 : qtp219578974-84 KubernetesCloud/new.jelly KubernetesCloud/config.jelly'
	at hudson.model.DescriptorVisibilityFilter.apply(DescriptorVisibilityFilter.java:73)
	at hudson.Functions.filterDescriptors(Functions.java:2186)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
```

Reply:
See as well https://ci.jenkins.io/job/Plugins/job/kubernetes-plugin/job/PR-1443/23/testReport/org.csanchez.jenkins.plugins.kubernetes/KubernetesCloudTest/defaultWorkspaceVolume/